Containerfile

```
@@ -6,8 +6,12 @@ RUN dnf -y install \
     python3-pip \
     && dnf clean all
 
+# Upgrade pip3
+RUN pip3 install --upgrade pip
+
 # Install ansible and ansible-lint
 RUN pip3 install --no-cache-dir \
```

I think we can reduce `RUN` steps by doing something like this
```suggestion
RUN pip3 install --upgrade pip && \
    pip3 install --no-cache-dir \
```